CHANGELOG.md

```diff
@@ -41,6 +43,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Disable PARANOX extension when using GEOS-Chem Classic 0.25x0.3125 or 0.125x0.15625 grids
 - Commented out met-fields `PEDGEDRY`, `PFICU`, `PFILSAN`, `PFLCU`, and `PFLLSAN` by default in GC-Classic and GCHP carbon HISTORY.rc, and GC-Classic CH4 HISTORY.rc
 - Turned on Carbon collection in `HISTORY.rc` for carbon simulations by default
+- Updated `run/shared/download_data.yml` so that aerosol and fullchem simulations will get the restart file from `GEOSCHEM_RESTARTS/GC_14.7.0`
+- Changed Extension 105 to be `DustL23M` instead of `DustDead` in `HEMCO_Config.rc.aerosol` and `HEMCO_Config.rc.fullchem`
```

Comment:
If the other dust extensions `DustDead` are no longer available, shouldn't they be removed to avoid confusion? If they should remain, then we should use a different extension number.